src/d3a/models/strategy/__init__.py

```diff
@@ -506,8 +511,8 @@ def post_bid(self, market, price, energy, replace_existing=True, buyer_origin=No
             energy,
             self.owner.name,
             original_bid_price=price,
-            buyer_origin=buyer_origin,
-            buyer_origin_id=buyer_origin_id,
+            buyer_origin=self.owner.name,
+            buyer_origin_id=self.owner.uuid,
```

Comment:
Great!
please remove `buyer_origin` and `buyer_origin_id` from the input parameters. AFAIS; they are not needed any more.